theme/static/js/resourceLandingAjax.js

```diff
@@ -1142,7 +1250,7 @@ function get_irods_folder_struct_ajax_submit(res_id, store_path) {
             bindFileBrowserItemEvents();
 
             $("#hs-file-browser").attr("data-current-path", store_path);
-            $("#upload-folder-path").text(store_path);
+            $("#upload-folder-path").text(store_path.replace("data/", "")); // We don't show the data folder in the UI path
```

Comment:
ah, here we go.  I say we don't show content as well and then update the server to no longer expect "data/contents" for file paths.

Reply:
I've looked at the backend and it's enforcing the path to start with data/contents...  Should be easy enough to remove that and just prefix the path with data/contents if it's not passed in.